package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "watch": "node scripts/watch.js index.d.ts",
+    "watch": "onchange **/*.d.ts -e out/**  -- npm run build",
```

Comment:
Remember to quote these arguments. Otherwise it's expanded by bash and watch becomes a little less useful (since it won't catch new files).